tests/end2end/test_public_reset_password.py

```diff
@@ -141,3 +141,42 @@ def test_sees_error_message_if_attempting_to_use_reset_code_twice(password_reset
     # Sees error.
     assert escape(_('Reset code "%(code)s" already used (%(isoformat)s)', code=password_reset.code,
                     isoformat=password_reset.modified_at.isoformat() + 'Z')) in res
+
+
+def test_sees_error_message_if_too_many_active_password_resets(db, testapp):
+    """Show error if too many active password resets exist."""
+    inactive_user = UserFactory(is_active=False)
+    db.session.commit()
+    assert inactive_user.is_active is False
+
+    # Create active password resets
+    for _i in range(0, PasswordReset.MAX_ALLOWED_ACTIVE_PASSWORD_RESETS):
+        # Goes to homepage.
+        res = testapp.get('/')
+        # Clicks on 'Forgot password'.
+        res = res.click(_('Forgot password?'))
+        # Fills out ForgotPasswordForm.
+        username_with_different_casing = inactive_user.email.upper()
```

Comment:
I think this "works irrespective of casing" logic has already been tested in other test case. It appears as clutter here, proposing to replace it with the simpler version used in many other tests:

```python
        # Fills out ForgotPasswordForm.
        form = res.forms['forgotPasswordForm']
        form['username'] = user.email
```

Reply:
:+1: